cmd/release-controller/http_changelog.go

```diff
@@ -32,6 +32,13 @@ func (c *Controller) renderChangeLog(w http.ResponseWriter, fromPull string, fro
 	}
 	ch := make(chan renderResult)
 
+	archExtension := ""
+	architecture := getArchitectureFromPullSpec(toPull)
```

Comment:
Probably not great to do this.  Nothing about pull spec requires architecture.  You should probably have this info come from the change log, not the pull spec, and that means changing `oc` to print info about what architecture is in the release.

Reply:
I have refactored the code to call the releaseInfo API to get the architecture.